executor/seqtest/main_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/util/testbridge"
+	"github.com/tikv/client-go/v2/config"
+	"go.uber.org/goleak"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
What does this function mean?

Reply:
You can take a look at Golang test document about [`TestMain`](https://pkg.go.dev/testing#hdr-Main).

Generally it is used to do package level setup logic for tests. We use it here to inject goleak verification and other setups.